TOOLS/mpv-osd-symbols.sfdir/README.md

```diff
@@ -0,0 +1,16 @@
+Steps to add new icons:
+
+- Install `fontforge`
+- Install the last freely licensed version of symbola (`ttf-symbola-free` in the
```

Comment:
I would remove this step from readme, source font for symbols can be different. You can mention in separate paragraph, that some glyphs in are extracted from symbola font.